catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/formulaeditor/FormulaEditorTest.java

```diff
@@ -71,17 +74,13 @@ public void numericValuesTest() {
 				.perform(click());
 
 		//typeText not working for formula editor, so use CustomActions.typeInValue
-		onFormulaEditor()
-				.performEnterNumber(12345.678);
+		onFormulaEditor().performEnterNumber(12345.678);
 		pressBack();
 	}
 
-	@After
-	public void tearDown() throws Exception {
```

Comment:
Please look into the other tests and make a proper teardown of the tests. (delete projects etc.)
Not an issue by created by you, but please fix it

Reply:
Opened ticket at https://jira.catrob.at/browse/IDE-191

This is out of scope for this PR. - It is large enough as is.